docs/openapi/components/schemas/common/MonthlyAdvanceManifest.yml

```diff
@@ -63,7 +82,11 @@ properties:
           numberOfBatches:
             type: string
             title: Number of Batches
-            description: Number of batches scheduled for the delivery location in one month       
+            description: Number of batches scheduled for the delivery location in one month 
+          countofShippers:
+            type: string
```

Comment:
Same as above, should this be type: number?

Reply:
Also, capital "O" in `countOfShippers`.

Reply:
Thanks @nissimsan  for catching this! I committed these changes